vars/npmRelease.groovy

```diff
@@ -10,12 +10,13 @@ def call(body) {
     def gitUserName = config.gitUserName ?: 'fabric8-release'
     def branch = config.branch
 
-    sh "git config user.email ${gitEmail}"
-    sh "git config user.name ${gitUserName}"
+    sh """
+       git config user.email ${gitEmail}
+       git config user.name ${gitUserName}
 
-    sh 'chmod 600 /root/.ssh-git/ssh-key'
-    sh 'chmod 600 /root/.ssh-git/ssh-key.pub'
-    sh 'chmod 700 /root/.ssh-git'
+       install -m 600 -D /root/.ssh-git-ro/ssh-key /root/.ssh-git/ssh-key
+       install -m 600 -D /root/.ssh-git-ro/ssh-key.pub /root/.ssh-git/ssh-key.pub
```

Comment:
Have you make sure that the directory auto created by install is going to be 700 ? should you add a chmod in there in case off ?

Reply:
actually nevermind, install does create by default as 700 : 

![image](https://user-images.githubusercontent.com/98980/38022464-d7dc28fa-327f-11e8-9849-ef35b3b510b4.png)